src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -41,6 +41,10 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 
 ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
+           # check library crates on all tier 1 targets
+           # We disable optimizing compiler built-ins because that requires building for the target
```

Comment:
```suggestion
           # We disable optimizing compiler built-ins because that requires building for the target.
```

Reply:
It looks like my updated suggestion that I edited in above hasn't been picked up in the latest patch.

Reply:
Oh, it didn't show me the edit until I refreshed. Ho hum.